src/components/avatar-image.tsx

```diff
@@ -26,6 +26,7 @@ export const AvatarImage: React.VoidFunctionComponent<AvatarImageProps> = ({
 }) => {
   return (
     <StyledImg
+      referrerPolicy="no-referrer"
```

Comment:
Avoids odd 403 for avatar image if it's coming from google

https://stackoverflow.com/a/61042200/1818285